compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -633,6 +643,9 @@ class MapAllocatorCache {
   u16 LRUTail GUARDED_BY(Mutex) = 0;
   // The AvailableHead is the top of the stack of available entries
   u16 AvailableHead GUARDED_BY(Mutex) = 0;
+  // The DecommitBase is the least recently used entry that has not
+  // been released
+  u16 DecommitBase GUARDED_BY(Mutex) = 0;
```

Comment:
`DecommitBase` gives me the idea of entry being released (decommitted). Can we change it to something like `LastUnreleasedEntry` or making it point to the first released entry?

Reply:
I think changing the name to `LastUnreleasedEntry` makes the most sense since this is the entry we always want to release. I've updated the code accordingly.